tests/projects/project1/.dockerignore

```diff
@@ -0,0 +1,10 @@
+/.git
+/.gomodcache
+/build/*
+!/build/rpms/
+/build/rpms/*
```

Comment:
I don't love copying this file around but I'll clean it up in #102 assuming this merges first.